app/views/users/dashboard.html.erb

```diff
@@ -0,0 +1,61 @@
+
+
+<%= link_to "Create a new category", new_category_path, class: "btn btn-primary"%>
+<%= link_to "Add a new product", new_product_path, class: "btn btn-primary" %>
+
+<% if flash.now[:error] %>
+  <div>
+    <%=flash.now[:error] %>
+  </div>
+<%end%>
+
+
+
+
+<h3>My Products</h3>
+<table class="table">
+  <tr>
+  <th>Product Name</th>
+  <th>Description</th>
+  <th>Price</th>
+  <th>Update</th>
+
+  </tr>
+<% if @products %>
+  <% @products.each do |product| %>
+  <tr>
+    <td><%= link_to "#{product.name}", product_path(product) %></td>
+    <td><%= product.description %></td>
+    <td><%= product.price %></td>
+    <td><%= link_to "Update Product", edit_product_path(product), class: "btn btn-info" %></td>
+    </tr>
+  <% end %>
+<% else %>
+  <p>You currently have no products for sale.</p>
+<% end %>
+</table>
+
+<hr>
+
+
+<h3>My Orders</h3><%= link_to "Order Fulfillment", dashboard_orders_path, class: "btn btn-primary" %>
+
+<table class="table">
+  <tr>
+    <th>Order Id</th>
+    <th>Order Status</th>
+    <th>Order Placed</th>
+    <th>View Order</th>
+  </tr>
+<% if @orders %>
+  <% @orders.each do |order| %>
+  <tr>
+    <td><%= order.id %></td>
+    <td><%= order.status.capitalize %></td>
+    <td><%= order.created_at.localtime.strftime("%B %-d, %Y %I:%M %p") %></td>
```

Comment:
I've seen this a few times throughout the app with formatting date/times. I think it would make sense to create a custom helper that would handle this dispaly.